src/App.tsx

```diff
@@ -1,10 +1,65 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import cn from 'classnames';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+// import { TodoList } from './TodoList';
 
-const USER_ID = 0;
+const USER_ID = 10995;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [err, setErr] = useState<string | null>(null);
```

Comment:
```suggestion
  const [error, setError] = useState<string | null>(null);
```
There is no need to shorten the names. It's not good for readability